source/class/qx/test/data/singlevalue/Array.js

```diff
@@ -100,8 +100,6 @@ qx.Class.define("qx.test.data.singlevalue.Array",
       });
       this.__label = new qx.test.data.singlevalue.TextFieldDummy();
 
-      // remove all bindings
```

Comment:
Is it safe to simply delete this? Can we do it because the tests don't run in one single execution context any more?

Reply:
removing off all bindings here is not necessary - test works without. Removing of all bindings is deprecated - so tests should'nt depent on it.

Reply:
Ok thanks!